MAUI/Rating/Appearance-Customization.md

```diff
@@ -0,0 +1,213 @@
+---
+
+layout: post
+title: Appearance Customization in .NET MAUI Rating control | Syncfusion
+description: Learn here all about Appearance Customization support in Syncfusion .NET MAUI Rating (SfRating) control and more.
+platform: maui
+control: Rating
+documentation: ug
+
+---
+
+# Appearance Customization in .NET MAUI Rating (SfRating)
+
+We can customize the rated and unrated fill color, stroke and stroke thickness using the following properties of [`RatingSettings`].
```

Comment:
You can customize the rated and unrated fill color, stroke, and stroke thickness using the following properties of [`RatingSettings`].

Reply:
Changed